src/cli.ts

```diff
@@ -95,36 +98,35 @@ async function runTests() {
     results.push(await NetScoreTest());
     apiRemaining.push((await OCTOKIT.rateLimit.get()).data.rate.remaining);
 
-    // Calc used rate limit 📝
     // Calc used rate limit 📝
     let usedRateLimit = apiRemaining[0] - apiRemaining[apiRemaining.length - 1];
-    console.log(`Rate Limit Usage:`);
-    console.log(`License Test: ${apiRemaining[0] - apiRemaining[1]}`);
-    console.log(`Bus Factor Test: ${apiRemaining[1] - apiRemaining[2]}`);
-    console.log(`Correctness Test: ${apiRemaining[2] - apiRemaining[3]}`);
-    console.log(`Ramp Up Test: ${apiRemaining[3] - apiRemaining[4]}`);
-    console.log(`Maintainability Test: ${apiRemaining[4] - apiRemaining[5]}`);
-    console.log(`Net Score Test: ${apiRemaining[5] - apiRemaining[6]}`);
-    console.log(`Total Rate Limit Used: ${usedRateLimit}`);
+    logger.debug(`Rate Limit Usage:`);
+    logger.debug(`License Test: ${apiRemaining[0] - apiRemaining[1]}`);
+    logger.debug(`Bus Factor Test: ${apiRemaining[1] - apiRemaining[2]}`);
+    logger.debug(`Correctness Test: ${apiRemaining[2] - apiRemaining[3]}`);
+    logger.debug(`Ramp Up Test: ${apiRemaining[3] - apiRemaining[4]}`);
+    logger.debug(`Maintainability Test: ${apiRemaining[4] - apiRemaining[5]}`);
+    logger.debug(`Net Score Test: ${apiRemaining[5] - apiRemaining[6]}`);
+    logger.debug(`Total Rate Limit Used: ${usedRateLimit}`);
 
     // Display test results
     results.forEach((result, index) => {
         passedTests += result.passed;
         failedTests += result.failed;
     });
 
-    console.log(`\x1b[1;32mTests Passed: ${passedTests}\x1b[0m`);
-    console.log(`\x1b[1;31mTests Failed: ${failedTests}\x1b[0m`);
-    console.log('\x1b[1;34mTests complete\x1b[0m');
+    process.stdout.write(`Tests Passed: ${passedTests}\n`);
+    process.stdout.write(`Tests Failed: ${failedTests}\n`);
+    process.stdout.write("Tests complete\n");
 
     // Syntax checker stuff (may move to run file in future idk)
     let coverage: number = Math.round(passedTests / (passedTests + failedTests) * 100); // dummy variable for now
     let total: number = passedTests + failedTests;
     
-    process.stdout.write(`Total: ${total}\n`);
-    process.stdout.write(`Passed: ${passedTests}\n`);
-    process.stdout.write(`Coverage: ${coverage}%\n`);
-    process.stdout.write(`${passedTests}/${total} test cases passed. ${coverage}% line coverage achieved.\n`);
+    logger.info(`Total: ${total}`);
+    logger.info(`Passed: ${passedTests}`);
+    logger.info(`Coverage: ${coverage}%`);
```

Comment:
why is this in info?? This is needed by the Checker... as seen in 

commit 611f8ccd2665ae08213f95a22b652fc8948a2914